paasta_tools/chronos_tools.py

```diff
@@ -597,6 +596,27 @@ def get_key(job):
     )
 
 
+def get_chronos_status_for_job(client, service, instance):
+    """
+    Returns the status (queued, idle, running, etc.) for a specific job from
+    the undocumented Chronos csv api that has the following format:
+
+    node,example_job,success,queued
+    node,parent_job,success,idle
+    node,child_job,success,idle
+    link,parent_job,child_job
+
+    :param client: Chronos client object
+    :param service: service name
+    :param instance: instance name
+    """
+    resp = client._call("/scheduler/graph/csv", "GET")
```

Comment:
I'm glad we have this, it provides nice data and I though we would need it eventually.

I'm a bit concerned about speed. In your testing doesn't this take quite a number of seconds to GET?